src/drawing/DrawingAnnotations.tsx

```diff
@@ -95,6 +96,8 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
         }
     }, [drawnPathGroups]);
 
+    const filterID = `ba-DrawingSVG-shadow_${uuid.v4()}`;
```

Comment:
I wonder if it would make more sense for this filterID to be generated by the `DrawingSVG` itself since it's the filter defined there on the svg. Maybe it could expose the filterID via context?

Reply:
Good idea, will implement.